beacon-chain/sync/service_test.go

```diff
@@ -0,0 +1,48 @@
+package sync
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/internal"
+)
+
+func TestStatus(t *testing.T) {
+
+	db := internal.SetupDB(t)
+	defer internal.TeardownDB(t, db)
+
+	if err := db.InitializeState(); err != nil {
+		t.Fatalf("Failed to initialize state: %v", err)
+	}
+
+	cfg := &Config{
+		ChainService:  &mockChainService{},
+		P2P:           &mockP2P{},
+		BeaconDB:      db,
+		AttestService: &mockAttestService{},
+	}
+
+	ss := NewSyncService(context.Background(), cfg)
+	synced, err := ss.Querier.IsSynced()
+
+	if err != nil {
+		if err1 := ss.Status(); err1 != err {
```

Comment:
Can we not use err1? Let's have a specific variable name there

Reply:
Hmmm... you mean to replace err1 by errStatus?